tests/plugins/test_store_metadata.py

```diff
@@ -90,15 +87,10 @@ def df_path(self):
         raise AttributeError("Dockerfile has not yet been generated")
 
 
-def prepare(pulp_registries=None, docker_registries=None, before_dockerfile=False,  # noqa
+def prepare(docker_registries=None, before_dockerfile=False,  # noqa
             reactor_config_map=False):
-    if pulp_registries is None:
-        pulp_registries = (
-            ("test", LOCALHOST_REGISTRY),
-        )
-
     if docker_registries is None:
-        docker_registries = (DOCKER0_REGISTRY,)
+        docker_registries = (LOCALHOST_REGISTRY, DOCKER0_REGISTRY,)
```

Comment:
That's a bit unexpected. Is that intentional? We weren't doing it before.

Reply:
the `pulp_registries` argument is effectively None, except the argument is getting removed.
Previously, when the `pulp_registries` argument was None, pulp_registries was set to ("test", LOCALHOST_REGISTRY) and that was added to push_conf.pulp_registries and push_conf.all_registries.  This is the minimal change to preserve that behavior.

The other option is to remove all of the LOCALHOST_REGISTRY entries from expected.  That's probably more correct, so I'll go ahead and do that.